blk/libblksharedqueue/Makefile

```diff
@@ -22,11 +22,11 @@ CFLAGS := \
 	-I$(SDDF)/include \
 	-I$(SDDF)/util/include
 
-all: $(BUILD_DIR)/sddf_block_shared_queue.o
```

Comment:
The object file should still retain the prefix to not clash with anything.

Reply:
It's unlikely, but this naming will be consistent with the other ring buffers etc